lib/ip_wrangler/iptables.rb

```diff
@@ -0,0 +1,227 @@
+module IpWrangler
+  class Iptables
+    $iptables_bin_path = '/usr/bin/sudo /sbin/iptables'
+    $awk_bin_path = '/usr/bin/awk'
+    $tail_bin_path = '/usr/bin/tail'
+    $grep_bin_path = '/bin/grep'
+
+    def initialize(chain_name, logger)
+      @chain_name = chain_name
+      @logger = logger
+    end
+
+    def rule_nat_port(public_ip, public_port, private_ip, private_port, protocol)
+      [Parameter.destination(public_ip),
+        Parameter.protocol(protocol),
+        Parameter.destination_port(public_port),
+        Parameter.jump('DNAT'),
+        Parameter.to_destination("#{private_ip}:#{private_port}")]
+    end
+
+    def rule_nat_ip(public_ip, private_ip)
+      rule_dnat = [Parameter.destination(public_ip),
+        Parameter.jump('DNAT'),
```

Comment:
Align the elements of an array literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.